python/chilldkg_ref/chilldkg.py

```diff
@@ -709,7 +731,15 @@ def coordinator_finalize(
 
 
 def coordinator_blame(pmsgs: List[ParticipantMsg1]) -> List[CoordinatorBlameMsg]:
-    """Perform the coordinator's blame step of a ChillDKG session. TODO"""
+    """Generate blame messages for participants to allow blame investigation.
```

Comment:
```suggestion
    """Generate investigation messages for a ChillDKG session.

    A participant's investigation message will allow the participants
    to investigate who is to blame for a failed ChillDKG session (see
    `participant_investigate`).
```

This keeps the first line short (and a single line) and adds an explanatory sentence instead.

Reply:
Done